packages/api/src/routes/medical/dtos/networkDTO.ts

```diff
@@ -0,0 +1,19 @@
+import { normalizeState } from "@metriport/shared";
+import { Network } from "../../../domain/medical/network";
+import { CQDirectoryEntry2 } from "../../../external/carequality/cq-directory";
+import { BaseDTO } from "./baseDTO";
+
+export type NetworkDTO = BaseDTO & Network;
+
+export function dtoFromHieDirectoryEntry(network: CQDirectoryEntry2): NetworkDTO {
+  return {
+    id: network.id,
+    eTag: "1",
+    name: network.name,
+    oid: network.id,
+    zip: network.zip,
+    state: network.state ? normalizeState(network.state) : undefined,
```

Comment:
Was there a reason why we left off the `addressLine` and `city` fields from the networks? Is it mainly that we think the networks wouldn't be happy that we're giving out their addresses?

Reply:
Was working off CW excel doc which doesnt have it. So as a v0 only wanted to include fields from both. Still need to test the CW api to see what they would return for their directory

Reply:
makes sense!